src/main.ts

```diff
@@ -9,9 +9,16 @@ import { parseInputs } from './utils'
  */
 export async function run(): Promise<void> {
   try {
-    const inputs = parseInputs()
-    const runner = new Runner(inputs.apiKey)
-    await runner.run(inputs)
+    const inputs: Inputs[] = parseInputs()
+
+    const runners: Promise<void>[] = inputs.map(async input => {
+      const runner = new Runner(input.apiKey)
+      return runner.run(input)
```

Comment:
What will happen if an input has an empty list of issues (e.g. you configure it with 2 teams, BE and DEVX, but only BE-XXX issue identifiers are passed in)?